broker-core/src/main/java/io/zeebe/broker/logstreams/processor/TypedEventStreamProcessorBuilder.java

```diff
@@ -109,6 +111,12 @@ public TypedEventStreamProcessorBuilder keyGenerator(KeyGenerator keyGenerator)
     return this;
   }
 
+  public TypedEventStreamProcessorBuilder withRocksDBController(
```

Comment:
Where do we do this? Or do you mean with `finalize`? Regardless, I thought about doing that, since we open it automatically when the stream processor is opened, so it's intuitive to do the same for closing I guess.

Reply:
No. But you could use the same listener as in `#withStateResource()`.

Reply:
Ah, I see it, yeah I'll add that.